tests/core/text/match/test_core_text_match.odin

```diff
@@ -25,7 +25,7 @@ failed :: proc(t: ^testing.T, ok: bool, loc := #caller_location) -> bool {
 expect :: testing.expect
 
 logf :: proc(t: ^testing.T, format: string, args: ..any) {
-	fmt.wprintf(t.w, format, ..args)
+	fmt.printf(/*t.w,*/ format, ..args)
```

Comment:
Why is this not using the testing writer and just using stdout?

Not a criticism, just a confusion.